comparison.dd

```diff
@@ -77,7 +77,7 @@ $(ITEMIZE
 		$(A spec/unittest.html, Unit testing),
 		$(A spec/module.html#staticorder, Static construction order),
 		$(A spec/statement.html#DeclarationStatement, Guaranteed initialization),
-		$(A memory.html#raii, RAII (automatic destructors)),
+		$(A https://wiki.dlang.org/Memory_Management#RAII_.28Resource_Acquisition_Is_Initialization.29, RAII (automatic destructors)),
```

Comment:
I am not a huge friend of how media wiki create its headings ids. Is there a way to get a nicer URL like https://wiki.dlang.org/Memory_Management#RAII ?